src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -230,20 +236,24 @@ export default App.extend(extend({
     });
     attachment.upload(file);
 
+    Radio.request('ws', 'add', file);
```

Comment:
`file` is going to be `undefined` here. I think we need to change `upload(file)` to a promise and then do something like:

```js
attachment.upload(file).then(file => {
  Radio.request('ws', 'add', file);
})
```